modules/importing-repositories/procedure-data-handoff-and-custom-workflow-design.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="full-data-handoff-and-custom-workflow-design"]
+
+= Data handoff and custom workflow design
+
+When you configure the Bulk Import plugin by setting the importAPI field to `scaffolder`, the Bulk Import Backend ensures all necessary context is passed directly to the Scaffolder API.
+
+Your control over input data:: The Orchestrator workflow receives the input data required to automate processing for every selected repository. The payload includes, at minimum, the complete list of repository URLs and the user identity.
```

Comment:
Clarification: the orchestrator stores the complete list, but for each repository the bulk import launches the scaffolder template separately and passes only the single repository’s data to that template run.
```suggestion
Your control over input data:: The Orchestrator workflow receives the input data required to automate processing for every selected repository. For each repository in that list, the bulk import launches the scaffolder template once and passes the data for that single repository to the template run.
```